src/crl.rs

```diff
@@ -50,7 +50,6 @@ pub trait CertRevocationList: Sealed {
 /// [^1]: <https://www.rfc-editor.org/rfc/rfc5280#section-5>
 #[cfg(feature = "alloc")]
 #[cfg_attr(docsrs, doc(cfg(feature = "alloc")))]
-#[allow(dead_code)] // we parse some fields we don't expose now, but may choose to expose in the future.
```

Comment:
Thanks! I think I also fixed this up in one of my cert CRL DP/CRL IDP branches.

Really wish there was a clippy lint that caught this :disappointed: